addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -132,4 +177,64 @@ export class CallParticipantCard extends Component {
             rtcSession: this.rtcSession,
         });
     }
+
+    onMouseDown() {
+        if (!this.isInset) {
+            return;
+        }
+        document.onmouseup = () => {
+            const insetEl = this.root.el;
+            if (parseInt(insetEl.style.left) < insetEl.parentNode.offsetWidth / 2) {
+                insetEl.style.left = "1vh";
+                insetEl.style.right = "";
+            } else {
+                insetEl.style.left = "";
+                insetEl.style.right = "1vh";
+            }
+            if (parseInt(insetEl.style.top) < insetEl.parentNode.offsetHeight / 2) {
+                insetEl.style.top = "1vh";
+                insetEl.style.bottom = "";
+            } else {
+                insetEl.style.bottom = "1vh";
+                insetEl.style.top = "";
+            }
+            document.onmouseup = null;
+            document.onmousemove = null;
+        };
+        document.onmousemove = (ev) => this.drag(ev);
+    }
+
+    drag(ev) {
```

Comment:
In mobile/chat-window it does not work very well (firefox/chrome) and the inset card covers most of the leave-call button.

Reply:
Can be ignored for now. This is not a critical bug, at least in chat window we can drag up/down, and even if it didn't work, it's best to consider that as known issues and fix later